app/screens/scheduled_post_options/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
0/5 really simple, but no tests for this file.

Reply:
Added to the list, can be taken up later.